test_cromwell.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+shutdown() {
+    cd $INITIAL_DIR
+    pkill -P $$
+    exit 0
+}
+
+trap "shutdown" EXIT
```

Comment:
Nice use of `trap`. In general I also like seeing `set -e` so the script doesn't just keep running on error.

http://www.alittlemadness.com/2012/06/25/bash-tip-reliable-clean-up-with-trap/